### PR TITLE
Adding nodejs package.json warning when using AzureFiles

### PR DESCRIPTION
**Pull request by alrod**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs [hunk 1]
```diff
@@ -608,5 +611,19 @@ public async Task<bool> RestartWorkerWithInvocationIdAsync(string invocationId)
             }
             return false;
         }
+
+        internal void CheckPackageJson()
+        {
+            if (_workerRuntime == RpcWorkerConstants.NodeLanguageWorkerName && _environment.AzureFilesAppSettingsExist())
```
Comment (safihamid) [inline src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:619]: Add a comment of our findings and why cold start could be affected if package.json is missing?

File: src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs [hunk 2]
```diff
@@ -608,5 +611,19 @@ public async Task<bool> RestartWorkerWithInvocationIdAsync(string invocationId)
...
+            {
+                Utility.ExecuteAfterColdStartDelay(_environment, () =>
+                {
+                    if (!File.Exists(Path.Combine(_scriptOptions.RootScriptPath, "package.json")))
+                    {
+                        _logger.LogDebug("package.json is not found in Azure Files - cold start for NodeJs can be affected.");
```
Comment (safihamid) [inline src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs]: update the message to: .... not found at the root of the Function App?

**Review by safihamid — APPROVED** [review]
:shipit:

File: src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs [hunk 3]
```diff
@@ -608,5 +611,21 @@ public async Task<bool> RestartWorkerWithInvocationIdAsync(string invocationId)
             }
             return false;
         }
+
+        internal void CheckPackageJson()
+        {
+            // On reading a js file with funciton code('require') NodeJs tries to find 'package.json' all the way up to the file system root.
```
Comment (liliankasem) [inline src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:617]:
```suggestion
            // On reading a js file with function code('require') NodeJs tries to find 'package.json' all the way up to the file system root.
```

**Review by fabiocav — CHANGES_REQUESTED** [review]
Why is this being added to the host and not logged by the worker? This check doesn't really belong here (even if done in the host, it shouldn't be in this class).

**Comment by alrod** [comment]
closing in favor:
https://github.com/Azure/azure-functions-nodejs-worker/pull/501

**Pull request closed without merging**